model-engine/model_engine_server/inference/batch_inference/vllm_batch.py

```diff
@@ -414,6 +420,30 @@ async def generate_with_vllm(
         if return_token_log_probs:
             output.tokens = tokens
 
+        tags = [
+            f"model:{model}",
+            f"use_tool:{use_tool}",
```

Comment:
Is use_tool actually used to control any behavior? Seems kind of funny to have a parameter that is not linked to any behavioral change.

Reply:
it's used to control whether we want to run python code